src/app/blog/[slug]/page.tsx

```diff
@@ -48,6 +49,7 @@ export default function BlogPost({ params }: BlogPostProps) {
           category={category}
         />
         {content && <MarkdownContent>{content}</MarkdownContent>}
+        <SharePost title="Share post" postTitle={title} />
```

Comment:
```suggestion
        <SharePost title="Share Post" postTitle={title} />
```

But also nice to add `capitalize` to `<p className="capitalize text-brand-300">{title}</p>`